bessctl/commands.py

```diff
@@ -833,9 +833,16 @@ def _do_run_file(cli, conf_file):
 
     try:
         code = compile(xformed, conf_file, 'exec')
-    except:
-        # TODO: Provide more information where and why fail to compile
-        cli.err('Fail to compile bess config file %s ' % conf_file)
+    except SyntaxError as e:
+        # TODO: e.offset might be wrong if there's a correct syntactic
+        #       sugar in an erroneous line
+
```

Comment:
Maybe just point out the line and not the column, since we don't have pre-sugar column info?  (Otherwise this looks good to me, and I don't really have a strong objection to having the column indicator)

Reply:
Somehow I tend to make mistakes in "sugar free" lines, probably because I connect modules with separate commands.  But I can rework the patch to only report the line number and show the erroneous line (which I think speeds up the debugging process).

Reply:
I don't think I really care either way 😺 It was just a thought about a way to handle the `TODO` without getting complicated...